fuel/src/main/kotlin/com/github/kittinunf/fuel/toolbox/HttpClient.kt

```diff
@@ -27,8 +27,10 @@ import kotlin.coroutines.resumeWithException
 import kotlin.coroutines.suspendCoroutine
 import kotlin.math.max
 
-internal class HttpClient(
+class HttpClient(
     private val proxy: Proxy? = null,
+    var stethoHookFactory: (() -> StethoHook)? = null,
+    var stethoHook: StethoHook? = null,
```

Comment:
is this is still needed since you going to assign it from the Factory?

Reply:
thanks! I removed it and make it a local variable.